backends/graphics/null/null-graphics.h

```diff
@@ -32,6 +32,7 @@ class NullGraphicsManager : public GraphicsManager {
 	void setFeatureState(OSystem::Feature f, bool enable) override {}
 	bool getFeatureState(OSystem::Feature f) const override { return false; }
 
+#ifdef USE_RGB_COLOR
```

Comment:
This must go to a separate commit.

Reply:
It already is: "COMMON: Fix null graphics backend without USE_RGB_COLOR" ... maybe you meant separate PR?